TESTS/mbed_drivers/mem_trace/main.cpp

```diff
@@ -391,7 +391,7 @@ static void test_case_multithread_malloc_free()
         threads[i].start(callback(malloc_free, &threads_continue));
     }
 
-    Thread::wait(wait_time_us);
+    ThisThread::sleep_for(wait_time_us);
```

Comment:
I note in passing that this takes milliseconds not microseconds. I wonder if this is supposed to run for 10 seconds or 10 milliseconds? Not changing it in this PR, anyway.